hummingbird/ml/operator_converters/_tree_commons.py

```diff
@@ -80,7 +80,7 @@ def __call__(self, x):
 class ApplySigmoidPostTransform(PostTransform):
     def __call__(self, x):
         output = torch.sigmoid(x)
-        return torch.cat([1 - output, output], dim=1)
+        return torch.cat([torch.tensor(1.0, dtype=output.dtype) - output, output], dim=1)
```

Comment:
Do we actually need to properly cast 1 or creating a tensor is enough?

Reply:
Just the tensor without dtype is sufficient. I made the change.